IFC4x3/PropertyConstants/s/SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol" Name="SLIPFORMED" UniqueId="4e592306-ad51-46b0-b920-28e0ff10262f" />
```

Comment:
Wiki: "Slip forming, continuous poured, continuously formed, or slipform construction is a construction method in which concrete is poured into a continuously moving form."
or
https://theconstructor.org/construction/slipform-construction-uses/185/: "The slipform helps to conduct continuous pouring of the concrete to the moving formwork."
```suggestion
<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol" Name="SLIPFORMED" UniqueId="4e592306-ad51-46b0-b920-28e0ff10262f" />
```

Reply:
Added description in [03a28cc](https://github.com/bSI-InfraRoom/IFC-Specification/pull/824/commits/03a28cc8c6229008d4d538438d999ed34110178f)